sky/jobs/state.py

```diff
@@ -1335,11 +1511,11 @@ def scheduler_set_alive_waiting(job_id: int) -> None:
 
 
 @_init_db
-def scheduler_set_done(job_id: int, idempotent: bool = False) -> None:
+def scheduler_set_done(job_id: int) -> None:
     """Do not call without holding the scheduler lock."""
```

Comment:
We can delete these comments now I think. There are some similar comments in other functions and in scheduler.py